Algebraic_foundations/doc/Algebraic_foundations/Concepts/EuclideanRing.h

```diff
@@ -21,7 +21,7 @@ Moreover, `CGAL::Algebraic_structure_traits< EuclideanRing >` is a model of
 
 \cgalHeading{Remarks}
 
-The most prominent example of a Euclidean ring are the integers.
+The most prominent example of an Euclidean ring are the integers.
```

Comment:
Not sure about this one.  See [this discussion](https://forum.wordreference.com/threads/a-an-euclidean-eu.1067647/)

Reply:
Did some checks as well looks indeed that in this case `a` is the right choice.